### PR TITLE
Added autoprefixer to build system

### PR DESCRIPTION
**Pull request by arulprabakaran**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added autoprifxing css into build system using postcss and autoprefixer (postcss plugin).

Fixes: #297

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -65,6 +67,9 @@ gulp.task('pug', () => {
 gulp.task('styl', () => {
 	return gulp.src('src/styl/index.styl')
 		.pipe(stylus({compress: true}))
+		.pipe(postcss([
+			autoprefixer({browsers: ['last 1 version']})
+    	]))
```
Comment (vikaspotluri123) [inline gulpfile.js]: Looks like there might be some spaces here... or github is messing with me

File: gulpfile.js [hunk 2]
```diff
@@ -65,6 +67,9 @@ gulp.task('pug', () => {
 gulp.task('styl', () => {
 	return gulp.src('src/styl/index.styl')
 		.pipe(stylus({compress: true}))
+		.pipe(postcss([
+			autoprefixer({browsers: ['last 1 version']})
```
Comment (vikaspotluri123) [inline gulpfile.js]: I think we should do at least `last 2 versions`

File: gulpfile.js (hunk 1 above)
Comment (arulprabakaran) [reply gulpfile.js]: It is fine in editor, maybe github compare make it look weird.

File: gulpfile.js (hunk 2 above)
Comment (arulprabakaran) [reply gulpfile.js]: Done

**Pull request merged**